conf/airframes/examples/bebop.xml

```diff
@@ -10,6 +10,17 @@
       <module name="udp"/>
     </target>
 
+    <define name="MT9F002_OUTPUT_HEIGHT" value="640" />
```

Comment:
These 3 sound good to me.
But it would be nicer to add those to the file where the defaults are defined.
So that the user can add something like
`<define name="BEBOP_CAMERA_CONFIG" value="HIGH_FPS"/>`
or similar.
If you omit that, you will get the defaults or you can define each option explicitly as done here...